tools/vsimporter/src/query.cpp

```diff
@@ -76,7 +76,7 @@ static bool parseListSelection(const String& selection, size_t upperBound, std::
 
 void queryListSelection(const StringVec& values, const String& queryMessage, const String& itemName, std::vector<size_t>& ret)
 {
-  sbAssert(!values.empty());
+  sbAssertWithTelemetry(!values.empty(), "Unable to query empty list of values");
```

Comment:
>sbAssertWithTelemetry(!values.empty(), "Unable to query empty list of values"); [](start = 2, length = 79)

Would the query message be helpful here?  I'm not sure if the query would contain Personally Identifiable Information (PII) or not.